### DISCUSSION TITLE
Problem installing ManimCE 0.11.0 on Linux Mint 20.1

### DISCUSSION THREAD

**Discussion by alexbottoni** [ManimCommunity/manim#2234] [General] [answered]
I just tried to install the current version of ManimCE (0.11.0) on my Linux Mint 20.1 Ulyssa, following official instructions (https://docs.manim.community/en/stable/installation/linux.html)

Installation process seems to go fine but when I try to execute the standard "first scene" example (as suggested here: https://docs.manim.community/en/stable/tutorials/quickstart.html#your-first-scene ), I get this error:

ImportError: /home/alex/anaconda3/lib/python3.7/site-packages/cairo/_cairo.cpython-37m-x86_64-linux-gnu.so: undefined symbol: cairo_svg_surface_set_document_unit

I found this article, that suggest to use pycairo 1.11.1:
https://www.reddit.com/r/Python/comments/doq8ye/installing_pycairo_issue/
Unfortunately, pycairo 1.11.1 is NOT compatible with ManimCE 0.11.0

So, I tried to install pycairo 1.19. I got this new error:
ImportError: /home/alex/anaconda3/lib/python3.7/site-packages/manimpango/cmanimpango.cpython-37m-x86_64-linux-gnu.so: undefined symbol: pango_layout_set_line_spacing

I also tried to install the whole stuff inside an Anaconda container and it did not work either.

How can I install ManimCE on Linux Mint? (I would like NOT to use a Docker container for experimenting with ManimCE).

**Comment by behackl** [comment, marked as answer]
Usually problems like that are caused by cairo being picked up from within anaconda instead of from the system itself, which tends to cause problems.

See [this answer on StackOverflow](https://stackoverflow.com/a/53972647) for instructions how to use `ldd` to see which version of cairo (or pango, in your second attempt) is actually used.

**Reply by alexbottoni** [reply]
Thanks. In order to leave my machine as it is, I decided to experiment with manim using a Linux Mint 20 virtual machine in Virtualbox. It works fine.
